### PR TITLE
EDSC-1770-b: Updated gibsUrl to new format

### PR DESCRIPTION
**Pull request by jboukes**
This PR updates the gibsURL value such that the projection is no longer included.

### CODE REVIEW

**Comment by mreese84** [comment]
I know there's no reviewers on this yet, but I took a look because I was curious. If you click on "map imagery" and load the first collection listed the JS console is throwing a bunch of "bad request" errors. It's trying to load imagery from https://gibs.earthdata.nasa.gov/wmts/epsg4326/best/MODIS_Aqua_Aerosol_Optical_Depth_3km/default/undefined/2km/2/0/0.png which says the time format is invalid.

**Comment by jboukes** [comment]
@mreese84 Thank you for finding that!  I will check it out.

**Comment by mreese84** [comment]
@jboukes so this is getting closer, a couple of my browser tests have passed. Notably, the LAADS collection we were looking at together and a random LP DAAC collection. The original collection listed in the ticket also works. So this is largely working.

However, when I load `C1236303843-NSIDC_ECS`no imagery displays and the JS console is _filled_ with 400 errors like we were seeing initially. When you follow one of the bad links it says `<ExceptionText>LAYER does not exist</ExceptionText>`. Take note that there is another NSIDC collection that _does_ work (`C179014689-NSIDC_ECS`).

Now, when I look at the same collection in Production I see the same errors. So before moving forward with this PR could you please look into what's going on there. Also, this build isn't green, so that needs to be resolved as well. But, I think we're really close.

File: app/assets/javascripts/modules/map/leaflet-plugins/granule_layer.js.coffee [hunk 1]
```diff
@@ -198,8 +198,10 @@ ns.GranuleLayer = do (L
         L.TileLayer.prototype.getTileUrl.call(this, tilePoint)
       else
         this._url = this._originalUrl || this._url;
-        L.TileLayer.prototype.getTileUrl.call(this, tilePoint) + "&time=#{date}" if matched
-
+        gibsUrl = L.TileLayer.prototype.getTileUrl.call(this, tilePoint)
+        return gibsUrl if gibsUrl.lastIndexOf('//') == -1
```
[the remark this answers is not in the document]
Comment (jboukes) [reply app/assets/javascripts/modules/map/leaflet-plugins/granule_layer.js.coffee]: @abbottry There is one scenario - test config :)  Test uses a local 'gibs-example.jpeg' call - presumably to cut down on external calls during test runs - and this does not include the '//'.

[the remark this answers is not in the document]
Comment (abbottry) [reply app/assets/javascripts/modules/map/leaflet-plugins/granule_layer.js.coffee]: Could you please add a comment about that?

[the remark this answers is not in the document]
Comment (jboukes) [reply app/assets/javascripts/modules/map/leaflet-plugins/granule_layer.js.coffee]: Sure 👍

**Pull request merged**